### DISCUSSION TITLE
9660 light sensor /custom bin/ - need to enter Sensor27 in console after each start?

### DISCUSSION THREAD

**Discussion by oernii** [arendst/Tasmota#14671] [Q&A] [answered]
Hi, I'd like to use the light sensor 9660, I have a working custom firmware, which works as described in https://tasmota.github.io/docs/APDS-9960/#configuration

My question: is there any way to permanently set "Sensor27" to 0, so that after powerup my device measures what I want?

PS: I had a look at the source code, do I need to modify the firmware for the default to change?
      disableGestureSensor();
      gesture_mode = 0;
      enableLightSensor();

**Comment by barbudor** [comment, marked as answer]
Indeed Sensor27 is not saved to flash Settings.
You can use a boot-time rule such as
```
Rule1 on System#Boot do Sensor27 0 endon
Rule1 1
```
There are a few boot triggers available depending when you want this to occur, see : https://tasmota.github.io/docs/Rules/#rule-trigger
`System#Boot` is the latest trigger that occur when everything has been initialized including WIfi and MQTT
You also have `Power1#Boot` or `Switch1#Boot` but they won't occur without a Relay or a Switch configured.
You can also use `Wifi#Connected` or `MQTT#Connected`
Choose the one of your liking 😂

If you have a filesystem, you can also use `autoexec.bat`

**Reply by oernii** [reply]
Thanks, exactly what I wanted to know.

**Reply by sfromis** [reply]
If you build a binary yourself, you could also config it in `user_config_override.h` defining `USE_APDS9960_STARTMODE`
